protocol.md

```diff
@@ -52,6 +53,17 @@ KEYID           | string | No       | No
     decisions; it may only be used to narrow the selection of possible keys to
     try.
 
+*   EXTENSION: Optional, unauthenticated object used to store signature-specific
+    information. Extensions are identified by a type field that unambiguously
+    describes the fields for the extension. The details for each extension's
+    type and its fields must be agreed upon out-of-band by the signer and
+    verifier, though some well-known extension types may be listed and defined
+    in the DSSE specification. Verifiers MUST use extension fields as a matter
```

Comment:
Do we want them listed in the specification or a separate page?  I would vote for something separate so we don't have to update the DSSE spec doc for any additions.  Adding an extension should be very lightweight and easy.

Reply:
It is a separate page, part of this PR. I've included a link.